setup/steps/check_pgss_available.go

```diff
@@ -0,0 +1,30 @@
+package steps
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/pganalyze/collector/setup/query"
+	s "github.com/pganalyze/collector/setup/state"
+)
+
+var CheckPgssAvailable = &s.Step{
+	Description: "Prepare for pg_stat_statements install",
+	Check: func(state *s.SetupState) (bool, error) {
+		row, err := state.QueryRunner.QueryRow(
+			fmt.Sprintf(
+				"SELECT true FROM pg_available_extensions WHERE name = 'pg_stat_statements'",
```

Comment:
We should fully schema-qualify this (and other similar queries), to avoid someone creating an object in the `public` schema (or maybe set search path?)

Reply:
Ended up setting search path before every query in `query/runner.go` (see ef8c5d2cf2363459351cabf9434d450cb7110582 once it's pushed)..